modules/Db2/scripts/uninstall_Db2.sh

```diff
@@ -0,0 +1,27 @@
+eval "$(jq -r '@sh "export KUBECONFIG=\(.kubeconfig) DB2_PROJECT_NAME=\(.db2_project_name)"')"
+
+ibmcloud login -q -apikey "${API_KEY}"
+ibmcloud ks cluster config -c "${CLUSTER_ID}" --admin
+
+echo "Deleting Resources"
```

Comment:
These are still resources for CP4I, not CP4BA

Reply:
Fixed